cvat-ui/src/components/job-item/job-item.tsx

```diff
@@ -113,14 +114,29 @@ function JobItem(props: Props): JSX.Element {
     }
     const frameCountPercent = ((job.frameCount / (task.size || 1)) * 100).toFixed(0);
     const frameCountPercentRepresentation = frameCountPercent === '0' ? '<1' : frameCountPercent;
+    let jobName = `Job #${job.id}`;
+    if (task.consensusJobsPerSegment && job.type !== JobType.GROUND_TRUTH) {
+        jobName = job.type === JobType.CONSENSUS ? `Consensus Job #${job.id}` : `Normal Job #${job.id}`;
```

Comment:
Do we really need `normal job` instead of just `job`?

Reply:
In the regular sync meet, when talking about jobs, to not get confused whether talking about consensus ones or the normal, we used to the "normal job", which swept in here as well. 

If you say I will change "Normal Job" to just "Job".

Reply:
Ok, lets leave it as is for now, its discussable I believe